src/components/App.js

```diff
@@ -1,243 +1,53 @@
-    const skill09 = getData('skills/investigation');
-    const skill10 = getData('skills/medicine');
-    const skill11 = getData('skills/nature');
-    const skill12 = getData('skills/perception');
-    const skill13 = getData('skills/performance');
-    const skill14 = getData('skills/persuasion');
-    const skill15 = getData('skills/religion');
-    const skill16 = getData('skills/sleight-of-hand');
-    const skill17 = getData('skills/stealth');
-    const skill18 = getData('skills/survival');
-
-    Promise.all([skill01, skill02, skill03, skill04, skill05, skill06, skill07, skill08, skill09, skill10, skill11, skill12, skill13, skill14, skill15, skill16, skill17, skill18])
-    .then(responses => {
-      if (updateSkills) {
-        responses.forEach(response => setSkills(prevState => [...prevState, response]))
-      }
-    })
-    .catch(error => setError(error.message))
-
-    return () => updateSkills = false;
-  }, [])
-
-  useEffect(() => {
-    let updateRaces = true;
-    const race01 = getData('races/dragonborn');
-    const race02 = getData('races/dwarf');
-    const race03 = getData('races/elf');
-    const race04 = getData('races/gnome');
-    const race05 = getData('races/half-elf');
-    const race06 = getData('races/half-orc');
-    const race07 = getData('races/halfling');
-    const race08 = getData('races/human');
-    const race09 = getData('races/tiefling');
-
-    Promise.all([race01, race02, race03, race04, race05, race06, race07, race08, race09])
-    .then(responses => {
-      if (updateRaces) {
-        responses.forEach(response => setRaces(prevState => [...prevState, response]))
-      }
-    })      
-    .catch(error => setError(error.message))
-
-    return () => updateRaces = false;
-  }, [])
-
-  useEffect(() => {
-    let updateClasses = true;
-    const charClass01 = getData('classes/barbarian');
-    const charClass02 = getData('classes/bard');
-    const charClass03 = getData('classes/cleric');
-    const charClass04 = getData('classes/druid');
-    const charClass05 = getData('classes/fighter');
-    const charClass06 = getData('classes/monk');
-    const charClass07 = getData('classes/paladin');
-    const charClass08 = getData('classes/ranger');
-    const charClass09 = getData('classes/rogue');
-    const charClass10 = getData('classes/sorcerer');
-    const charClass11 = getData('classes/warlock');
-    const charClass12 = getData('classes/wizard');
-
-    Promise.all([charClass01, charClass02, charClass03, charClass04, charClass05, charClass06, charClass07, charClass08, charClass09, charClass10, charClass11, charClass12])
-    .then(responses => {
-      if (updateClasses) {
-        responses.forEach(response => setClasses(prevState => [...prevState, response]))
-      }
-    })
-    .catch(error => setError(error.message))
-
-    return () => updateClasses = false;
-  }, [])
-
-  const updateRule = e => {
-    setRule(rules.find(rule => rule.index === e.target.id))
+  const NAMES = {
```

Comment:
naming convention for `NAMES` should be camel case? `names` or `gameNames`?

Reply:
I recently came across this from Web Dev Simplified that global variables that will remain unchanged should be in all caps. I still find this advice dubious :) But that's where I got the info. Need to do more research.